Inspicio/Controllers/ImagesController.cs

```diff
@@ -107,29 +108,51 @@ public class DataToBody
                 return NotFound();
             }
 
+            // Jack Lloyd 
+            // Fetch the image by the id pass in '/5'
             var image = await _context.Images.SingleOrDefaultAsync(m => m.ImageID == id);
             if (image == null)
             {
                 return NotFound();
             }
 
-            DataToBody DataToBody = new DataToBody();
-            DataToBody.Image = image;
+            // Jack Lloyd 
+            // Create a new FullReview object
+            // This will be passed into the view as the model.
+            FullReview FullReview = new FullReview();
+            FullReview.Image = image;
 
+            // Jack Lloyd 
+            // Fetch all the comments 
+            // This may not be the best way to do this...
             var allcomments = _context.Comments;
-            List<Comment> comments = new List<Comment>();
+
+            // Jack Lloyd 
+            // The FullReview comments holds a tuple holding one comment and one username.
```

Comment:
If you have the option of "saying" something with a comment vs the type system, I'd prefer to go with the type system. It'll check stuff for you at build time, and it can't get out of date when the code changes (and the code changes a lot when you're refactoring as you go)